elasticsearch/Dockerfile.centos7

```diff
@@ -60,7 +60,8 @@ ADD run.sh prep-install.${RELEASE_STREAM} install.sh ${HOME}/
 COPY utils/** /usr/local/bin/
 RUN ln -s /usr/local/bin/logging ${HOME}/logging && \
     ${HOME}/install.sh && \
-    rm -rf /tmp/lib
+    rm -rf /tmp/lib && \
+    echo "vm.max_map_count=262144" > /etc/sysctl.conf
```

Comment:
Do we need a similar change in elasticsearch/Dockerfile?

Reply:
I need to remove this line.. I was testing to see if that would make a difference and it doesn't. The way I was able to resolve this was doing that on my actual OCP nodes... so we need to investigate the best course of action when it comes to ensuring that is correctly set. If it is not at least that value ES 5 will not start up.

Reply:
Continuing the discussion...
So I asked around and it seems there is a way to modify the sysctl settings of pods using annotations [1] however updating a `vm` setting is not allowed by default and would actually make it so the entire cluster could use unsafe annotations.

The `openshift_node` has a task that updates the `sysctl`, so that may be the best course of action if one is going to deploy logging with es5.

[1] https://kubernetes.io/docs/concepts/cluster-administration/sysctl-cluster/

Reply:
https://github.com/openshift/openshift-ansible/pull/6628/commits/283e1902d2ec95ea3c6b369e9da234d5e13b02f7